sirepo/package_data/static/js/radia.js

```diff
@@ -30,24 +30,33 @@ SIREPO.app.config(function() {
             <div data-group-editor="" data-field="model[field]" data-model="model"></div>
         </div>
         <div data-ng-switch-when="HMFile" data-ng-class="fieldClass">
-            <div data-file-field="field" data-form="form" data-model="model" data-model-name="modelName"  data-selection-required="info[2]" data-empty-selection-text="No File Selected" data-file-type="h-m"></div>
+            <div data-file-field="field" data-form="form" data-model="model" data-model-name="modelName"  data-selection-required="false" data-empty-selection-text="No File Selected" data-file-type="h-m"></div>
         </div>
         <div data-ng-switch-when="IntStringArray" class="col-sm-7">
             <div data-number-list="" data-model="model" data-field="model[field]" data-info="info" data-type="Integer" data-count=""></div>
         </div>
         <div data-ng-switch-when="ObjectType" class="col-sm-7">
-            <div data-shape-selector="" data-model-name="modelName" data-model="model" data-field="field" data-field-class="fieldClass" data-parent-controller="parentController" data-view-name="viewName" data-object="viewLogic.getBaseObject()"></div>
+            <div data-shape-selector="" data-model-name="modelName" data-model="model" data-field="model[field]" data-field-class="fieldClass" data-parent-controller="parentController" data-view-name="viewName" data-object="viewLogic.getBaseObject()"></div>
         </div>
         <div data-ng-switch-when="MaterialType" data-ng-class="fieldClass">
-            '<select number-to-string class="form-control" data-ng-model="model[field]" data-ng-options="item[0] as item[1] for item in enum[info[1]]"></select>
+          <select number-to-string class="form-control" data-ng-model="model[field]" data-ng-options="item[0] as item[1] for item in enum[info[1]]"></select>
             <div class="sr-input-warning">
             </div>
         </div>
         <div data-ng-switch-when="PtsFile" data-ng-class="fieldClass">
-          '<input id="radia-pts-file-import" type="file" data-file-model="model[field]" accept=".dat,.txt"/>
+          <input id="radia-pts-file-import" type="file" data-file-model="model[field]" accept=".dat,.txt,.csv"/>
+        </div>
+        <div data-ng-switch-when="Points" data-ng-class="fieldClass">
+          <label class="control-label col-sm-5" style="text-align: center">{{ model.widthAxis }}</label> <label class="control-label col-sm-5"  style="text-align: center">{{ model.heightAxis }}</label>
+          <div class="col-sm-12" style="height: 200px; overflow-y: scroll; overflow-x: hidden;">
+              <div data-ng-repeat="p in model[field]">
+                <input class="form-control sr-number-list" data-string-to-number="float" data-ng-model="p[0]" data-ng-disabled="model.pointsFile" style="text-align: right;" required />
```

Comment:
does p have more than 2 elems? Maybe could be one input
```html
<input ... data-ng-repeat="e in p" data-ng-model="e" .../>

Reply:
[a47c51d](https://github.com/radiasoft/sirepo/pull/4686/commits/a47c51dfad824e6f5f04f57a903dcafc32d704ab)

Had to include `track by $index` to handle duplicate values